scripts/ena_project_biostudies.py

```diff
@@ -0,0 +1,166 @@
+# !/usr/bin/env python3
+
+# Copyright [2021] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests, xmltodict
+import argparse
+import pandas as pd
+from datetime import datetime
+
+description = """
+Description
+-----
+This script is required for the process of assigning DOI's to ENA projects, via the registration of a Biostudies record.
+
+This script outputs a pagetab file for the (currently) manual creation of a biostudies record, which can then be assigned a DOI via contacting the Biostudies team.
+This pagetab file/biostudies record can link to a single or multiple projects. 
+
+"""
+
+usage = """
+Usage: ena_project_biostudies.py <ARGUMENTS>
+Arguments:
+    --project         : (mandatory) name of main ENA project to link to Biostudies record
+    --additional_projects   : (optional) additional project accessions to link to created Biostudies entry/ single DOI (default: none)
+    --author : (mandatory) author names to include
+    --email : (mandatory) author email addresses
+    --institution : (mandatory) institution name
+    --output_file : (optional) output file name without extension. (default: timestamp)
+"""
+
+example = """
+Example: create pagetab file for a single project with one author
+    ena_project_biostudies.py --project PRJEB37886 --author 'Zahra Waheed' --email zahra@ebi.ac.uk --institution EBI --output_file test_May_25

```

Comment:
You're opening yourself up to spam by including your own email here 😅  
Let's add a fake person/address. Also, we can show that the output file can be a full filepath.
```suggestion
    ena_project_biostudies.py --project PRJEB37886 --author 'Joe Bloggs' --email joe@bloggs.com --institution EBI --output_file /path/to/output_file

```